pages/common/histexpand.md

```diff
@@ -0,0 +1,36 @@
+# history expansion
+
+> Reuse and expand the shell history in bash or zsh.
+> More information: <https://www.gnu.org/software/bash/manual/html_node/History-Interaction.html>.
+
+- Run the previous command:
+
+`!!`
+
+- Run the previous command as root:
+
+`sudo !!`
+
+- Run a command with the last argument of the previous command:
+
+`{{command}} !$`
+
+- Run a command with the first argument of the previous command:
+
+`{{command}} !^`
+
+- Run the command `n` lines back in the history:
+
+`!-{{n}}`
+
+- Run the most recent command starting with `string`:
+
+`!{{string}}`
+
+- Run the previous command, replacing `string1` with `string2`:
+
+`^{{string1}}^{{string2}}^`
+
+- Print the command `n` lines back without running it:
+
+`!-{{n}}:p`
```

Comment:
Maybe something like this? (in response to https://github.com/tldr-pages/tldr/pull/5455#discussion_r606831185)
```suggestion
- Perform a history expansion, but print the command that would be run instead of actually running it:

`{{!-n}}:p`
```